pynicotine/gtkgui/widgets/trayicon.py

```diff
@@ -232,7 +232,7 @@ def on_get_a_users_info(self, *_args):
 
         EntryDialog(
             parent=self.application.window,
-            title=_("Request User Info"),
+            title=_("View User Profile"),
             message=_('Enter the name of the user whose info you want to see:'),
```

Comment:
`'Enter the name of the user whose info you want to see:'`
 -> 
`'Enter the name of the user whose profile you want to see:'`